src/api/.rubocop_todo.yml

```diff
@@ -168,6 +168,7 @@ Metrics/ClassLength:
     - 'app/controllers/webui/search_controller.rb'
     - 'app/controllers/webui/staging/workflows_controller.rb'
     - 'app/controllers/webui/users_controller.rb'
+    - 'app/controllers/webui/users/tokens_controller.rb'
```

Comment:
This is needed because of one extra line?

Reply:
Apparently not. It was before with few more lines, but now it doesn't. Good catch. Dropped